<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdventureWindow</class>
 <widget class="QMainWindow" name="AdventureWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="blue_frame_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>551</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/pet_frame);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blank_bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>901</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/blank_bg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/card_01);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blank_bg_2">
    <property name="geometry">
     <rect>
      <x>-150</x>
      <y>530</y>
      <width>1181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/blank_bg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blank_bg_3">
    <property name="geometry">
     <rect>
      <x>-110</x>
      <y>-20</y>
      <width>1651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/blank_bg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/card_01);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/card_01);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>390</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/card_01);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mycard_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/card_01);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vscard_1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>-20</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/card_bg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vscard_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>-20</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/card_bg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vscard_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>-20</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/card_bg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vscard_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>-20</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/card_bg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vscard_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>-20</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/card_bg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="jjc_bg_blue">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/jjc_bg_blue);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="jjc_bg_grey">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>-30</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/jjc_bg_grey);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="jjc_bg_grey_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/jjc_bg_grey);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="jjc_bg_grey_3">
    <property name="geometry">
     <rect>
      <x>-100</x>
      <y>280</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/jjc_bg_grey);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="jjc_bg_grey_4">
    <property name="geometry">
     <rect>
      <x>-100</x>
      <y>-20</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/jjc_bg_grey);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="jjc_bg_grey_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/jjc_bg_grey);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="jjc_bg_grey_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>-170</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/jjc_bg_grey);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blue_frame">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>131</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/card_frame);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>131</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/card_frame);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blank_bg_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>291</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/blank_bg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blank_bg_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>61</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/blank_bg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="return_button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/pet_frame);</string>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card_button_1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>31</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="card_button_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>31</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="card_button_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>31</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="card_button_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>31</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="card_button_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="discard">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>410</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正像素24</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/pet_frame);</string>
    </property>
    <property name="text">
     <string>出牌</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>490</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正像素24</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>技能：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>170</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正像素24</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正像素24</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>380</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="vsname">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正像素24</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="myname">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>260</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正像素24</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="enemyCard_dis">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>111</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="myCard_dis">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>111</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>blank_bg</zorder>
   <zorder>jjc_bg_blue</zorder>
   <zorder>jjc_bg_grey_5</zorder>
   <zorder>jjc_bg_grey_6</zorder>
   <zorder>mycard_1</zorder>
   <zorder>mycard_2</zorder>
   <zorder>mycard_3</zorder>
   <zorder>mycard_4</zorder>
   <zorder>mycard_5</zorder>
   <zorder>vscard_1</zorder>
   <zorder>vscard_2</zorder>
   <zorder>vscard_3</zorder>
   <zorder>vscard_4</zorder>
   <zorder>vscard_5</zorder>
   <zorder>jjc_bg_grey</zorder>
   <zorder>jjc_bg_grey_2</zorder>
   <zorder>jjc_bg_grey_3</zorder>
   <zorder>jjc_bg_grey_4</zorder>
   <zorder>blue_frame</zorder>
   <zorder>label_2</zorder>
   <zorder>blue_frame_2</zorder>
   <zorder>blank_bg_2</zorder>
   <zorder>blank_bg_3</zorder>
   <zorder>blank_bg_4</zorder>
   <zorder>blank_bg_5</zorder>
   <zorder>return_button</zorder>
   <zorder>card_button_1</zorder>
   <zorder>card_button_2</zorder>
   <zorder>card_button_3</zorder>
   <zorder>card_button_4</zorder>
   <zorder>card_button_5</zorder>
   <zorder>discard</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>vsname</zorder>
   <zorder>myname</zorder>
   <zorder>enemyCard_dis</zorder>
   <zorder>myCard_dis</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
